java2/Makefile

```diff
@@ -1,5 +1,12 @@
 %.class: src/mal/
 	@rm -rf $*
+	@test -d bin || mkdir -p bin 
+	@if [ ! -f bin/jline-2.13.jar ]; \
```

Comment:
This could be a target/prerequisite to another one.

Reply:
thanks,  it fixed.